inbm/dispatcher-agent/dispatcher/schedule/sqlite_manager.py

```diff
@@ -41,7 +41,58 @@ def __init__(self, db_file=UDM_DB_FILE) -> None:
             raise DispatcherException(f"Error creating cursor: {e}")
         
         self._create_tables_if_not_exist() 
-            
+
+    def get_single_schedule_manifest(self) -> List[Tuple[int, int, int]]:
+        """
+        Get the data from single_schedule_manifest table and arrange them in ascending order based on the priority.
+        @return: List of tuple which represents priority, schedule_id and manifest_id.
+        """
+        try:
+            sql = ''' SELECT * FROM single_schedule_manifest; '''
```

Comment:
You can do this and it will still work even if column are added later:

```
sql = ''' SELECT priority, schedule_id, manifest_id FROM single_schedule_manifest; '''
```

Additionally, you can add `ORDER BY priority ASC` to have sqlite sort by priority rather than sort after you get the rows

Reply:
This is a good information!